File: Técnicas De Programação E Algoritmos/Switch case do primeiro dia do Gilmar/Semana.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.semana;
import java.util.Scanner;
/**
 *
 * @author Guilherme
 */
public class Semana {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        //Ler uma linha de texto, e pedir uma linha de texto
         System.out.println("Digite um numero para me dizer qual o dia da semana");
         int Semana = scanner.nextInt ();
         
         switch (Semana) {
            case 1:
                 System.out.println("O dia da semana e: Domingo");
                 break;
            case 2:
                System.out.println("O dia da semana é: Segunda");
                break;
            case 3:
                System.out.println("O dia da semana é: Terça");
                break;
            case 4:
                System.out.println("O dia da semana é: Quarta");
                break;
            case 5:
                System.out.println("O dia da semana é: Quinta");
                break;
            case 6:
                System.out.println("O dia da semana é: Sexta");
                break;
            case 7:
                System.out.println("O dai da semana é: Sabado");
                break;
            
            default:
                System.out.println("O numero digitado não corresponde a nenhum dia");
                break;
         }
         
    }
}

File: Programação Web/Atividade – JavaScript (Funções e Lógica de Programação)/Semana/semana.js

function nomeDoDia() {

    const semana = Number(document.getElementById("semana").value);
    const res = document.getElementById("res");

    switch (semana) {
        case 1:
            res.innerText = "Domingo";
            res.style.color = "goldenrod";
            break;
        case 2:
            res.innerText = "Segunda";
            res.style.color = "red";
            break;
        case 3:
            res.innerText = "Terça";
            res.style.color = "red";
            break;
        case 4:
            res.innerText = "Quarta";
            res.style.color = "goldenrod";
            break;
        case 5:
            res.innerText = "Quinta";
            res.style.color = "orange";
            break;
        case 6:
            res.innerText = "Sexta";
            res.style.color = "green";
            break;
        case 7:
            res.innerText = "Sabado";
            res.style.color = "green";
            break;
        default:
            res.innerText = "O número digitado não corresponde a nenhum dia da semana ❌";
            res.style.color = "red";
            break;
    }

    
}